### ISSUE TITLE
[Feature Request] An option for webhook to only be sent for NEW events

### ISSUE THREAD

**Issue by swemoney** [AHAAAAAAA/PokemonGo-Map#2538]
## Expected Behavior
The webhook should have an option that allows it to open be sent when a **new** event is triggered.

## Current Behavior
The webhook gets triggered for every event, even if the previous scan loop found that same pokemon.

## Possible Solution
Possibly check the "encounter_id" against the database to see if this event has already been encountered and then check for an option (webhook_only_new?) and ignore sending the webhook if both are true.

## Steps to Reproduce (for bugs)
1. Provide a webhook URL
2. Start a searcher
3. Watch as your webhook server gets pounded with repeated data

## Context
I'm working on using the webhooks to attach notifications to things (like a simple push notification for my phone or a large red siren light attached to a raspberry pi that lights up when a super rare pokemon is nearby). It's more of a hassle to maintain and check after the webhook is sent when it could so easily be added before it's ever sent out. Also saves on traffic across the web/network.

**Comment by LucasAdam** [comment]
I couldn't agree more ! One other option would be to do that on our end. Storing & recycling pokemons as they expire. And then we could avoid sending the same pokemon event twice.

**Comment by kvangent** [comment]
@swemoney I think you are better off doing it on your end so your app works with multiple instances of the server. I have a PokeAlarm implemented that handles it easily.

**Comment by tripout** [comment]
I was thinking about this ans storing this is a DB with the "encouter_id" and check if it's the same, if it's not push a new notification to the webhook.  Just an idea....

**Comment by swemoney** [comment]
I agree it isn't a mandatory enhancement or anything and I didn't think about issues with multiple searchers. I was pretty easily able to get around it with a simple Redis server that stored the encounter_id for a half hour so I could check if I'd received it or not. That's mainly why I'm proposing it as an option since I'm sure there are plenty of use cases for wanting every event to ping your callback server.
